src/libos/src/misc/uname.rs

```diff
@@ -48,3 +50,43 @@ fn copy_from_cstr_to_u8_array(src: &CStr, dst: &mut [u8]) {
     dst[..len].copy_from_slice(&src[..len]);
     dst[len] = 0;
 }
+
+fn obtain_nodename(dst: &mut [u8]) {
```

Comment:
Is it possible that we get store the hostname in parse_host_file(HostFile::HOSTNAME, hostname_ptr)? The current behavior is: get the hostname from host → parse the file and then save it as a file in SEFS → read the file to get the hostname. Is it too complex?

Reply:
Done